src/services/bria/index.ts

```diff
@@ -161,21 +170,103 @@ export const NewOnChainService = (): INewOnChainService => {
     }
   }
 
+  const findPayoutByRequestId = async (
+    requestId: PayoutRequestId,
+  ): Promise<PayoutId | OnChainServiceError> => {
+    try {
+      const request = new FindPayoutByExternalIdRequest()
+      request.setExternalId(requestId)
+
+      const response = await findPayoutByExternalId(request, metadata)
+      const foundPayout = response.getPayout()
+
+      if (foundPayout === undefined) return new PayoutNotFoundError()
+      return foundPayout.getPayoutQueueId() as PayoutId
+    } catch (err) {
+      if (err.code == status.NOT_FOUND) {
+        return new PayoutNotFoundError()
+      }
+      const errMsg = typeof err === "string" ? err : err.message
+      return new UnknownOnChainServiceError(errMsg)
+    }
+  }
+
+  const queuePayoutToAddress = async ({
+    walletId,
+    address,
+    amount,
+    speed,
+    requestId,
+  }: QueuePayoutToAddressArgs): Promise<PayoutId | OnChainServiceError> => {
+    try {
+      const request = new SubmitPayoutRequest()
+      request.setWalletName(briaConfig.walletName)
+      request.setPayoutQueueName(queueNameForSpeed(speed))
+      request.setOnchainAddress(address)
+      request.setSatoshis(Number(amount.amount))
+      if (requestId) {
+        request.setExternalId(requestId)
+        request.setMetadata(constructMetadata({ galoy: { walletId } }))
+      }
+
+      const response = await submitPayout(request, metadata)
+
+      return response.getId() as PayoutId
+    } catch (error) {
```

Comment:
handle status permission denied (ie. sending to blocked address)
error message: `DestinationBlocked - sending to '{0}' is prohibited`